ClickHouseClient/conf.ini

```diff
@@ -0,0 +1,3 @@
+[clickhouse]
```

Comment:
this seems to be your local files. please remove them from merge request and leave only required functionality